src/java/org/apache/cassandra/utils/concurrent/Ref.java

```diff
@@ -99,7 +100,8 @@
 public final class Ref<T> implements RefCounted<T>
 {
     static final Logger logger = LoggerFactory.getLogger(Ref.class);
-    public static final boolean DEBUG_ENABLED = TEST_DEBUG_REF_COUNT.getBoolean();
+    public static final boolean TRACE_ENABLED = TEST_DEBUG_REF_COUNT.getBoolean();
```

Comment:
can we still back to `DEBUG_ENABLED`?  the property is `debug`, and we add debug details... I don't personally see why we should change to `trace` instead?  you added a `DEBUG_EVENTS_ENABLED` but don't see that requiring us to rename this property.

Reply:
It’s just there’s two kinds of debugging now. Don’t feel super strongly about how we distinguish between them, but wanted to disambiguate and the old (now trace) logic traces call sites and stack traces, whereas the new permits you to log arbitrary status updates - this is what allowed me to quickly pin down the cache reference leak.

Reply:
I am ok with this change, but could cause rebases to be more annoying down the line.  I feel `debug` makes more sense than `trace` but will leave up to you